model/user.go

```diff
@@ -173,3 +174,56 @@ func GetUserScore(username string) (score UserScore, err error) {
 	`, username).Scan(&score.Score)
 	return
 }
+
+// Friend holds a friend
+type Friend struct {
+	Username   string `json:"username"`
+	CreateTime string `json:"createTime"`
```

Comment:
`CreateTime` should be of type `time.Time`, not `string`.

Reply:
Are you sure, i had time.Time first but you told me the DB returns only strings?
If i recall correctly 🤷‍♂️ .

Reply:
It's strings in the frontend, I believe time.Time is used elsewhere?